manager/integration/tests/common.py

```diff
@@ -807,3 +811,56 @@ def get_upgrade_test_image(cli_v, cli_minv,
                                      cli_v, cli_minv,
                                      ctl_v, ctl_minv,
                                      data_v, data_minv)
+
+
+def prepare_host_disk(dev, vol_name):
+    cmd = ['mkfs.ext4', dev]
+    subprocess.check_call(cmd)
+
+    mount_path = DIRECTORY_PATH + vol_name
+    # create directory before mount
+    cmd = ['mkdir', '-p', mount_path]
+    subprocess.check_call(cmd)
+
+    cmd = ['mount', dev, mount_path]
+    subprocess.check_call(cmd)
+    return mount_path
+
+
+def cleanup_host_disk_directory():
+    dir_list = os.listdir(DIRECTORY_PATH)
+    for dir in dir_list:
+        mount_path = DIRECTORY_PATH + dir
+        cmd = ['rm', '-r', mount_path]
+        subprocess.check_call(cmd)
+
+
+def umount_host_disk(vol_name):
+    mount_path = DIRECTORY_PATH + vol_name
+    cmd = ['umount', mount_path]
+    subprocess.check_call(cmd)
+
+
+def wait_for_volume_condition(client, name):
```

Comment:
Don't quite make sense to just wait for a condition to show up, since there can be multiple conditions and you may not got the one you're waiting for.

Add conditionType ("scheduled") and conditionStatus("true/false") to the parameter as well.